docs/source/getting_started/installation.rst

```diff
@@ -142,7 +142,7 @@ shimming-toolbox
 Development version
 ~~~~~~~~~~~~~~~~~~~
 
-Ensure that you have ``dcm2niix`` installed on your system.
+Ensure that you have ``dcm2niix`` > v1.0.20200331 installed on your system.
```

Comment:
I believe you mean greater than 20201102

Reply:
Fixing this one.

Reply:
Updated this to reflect changes of ```README.md``` as well